rel/app/macos/Sources/Livebook/Livebook.swift

```diff
@@ -53,7 +53,12 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 
         statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
         let button = statusItem.button!
-        button.image = NSImage(named: "MenuBarIcon")
+        let icon = NSImage(named: "LivebookIcon")!
+        let resizedIcon = NSImage(size: NSSize(width: 18, height: 18), flipped: false) { (dstRect) -> Bool in
+            icon.draw(in: dstRect)
+            return true
+        }
```

Comment:
we can also do this:

```suggestion
        }
        resizedIcon.isTemplate = true
```

which will follow the guidelines:



<img width="124" alt="image" src="https://user-images.githubusercontent.com/76071/225385434-8ab2465b-08aa-4259-9c1b-9ca00bf782da.png">

<img width="118" alt="image" src="https://user-images.githubusercontent.com/76071/225385506-63cab72d-3749-48eb-9db0-66aae2033f91.png">

but tbh I'd really miss the colourful icon! 😅 

cc @hugobarauna

Reply:
I prefer the colorful one, I agree!

Reply:
Let's keep the colours, can always revisit!